nix_update/__init__.py

```diff
@@ -84,16 +84,18 @@ def git_has_diff(git_dir: str, package: Package) -> bool:
 
 def git_commit(git_dir: str, attribute: str, package: Package) -> None:
     new_version = package.new_version
-    if new_version and package.old_version != new_version:
-        if new_version.startswith("v"):
-            new_version = new_version[1:]
-        msg = f"{attribute}: {package.old_version} -> {new_version}"
+    if new_version and package.old_version != new_version.number:
+        if new_version.number.startswith("v"):
+            new_version.number = new_version.number[1:]
+        msg = f"{attribute}: {package.old_version} -> {new_version.number}"
```

Comment:
What is returned for ``new_version``? I think it would be easier if that would just return the content of .number.

Reply:
A `nix_update.version.version.Version` is used everywhere to pass around the rev and make it consistent.